devtools-integration-tests/src/test/java/org/ballerina/devtools/debug/ModuleBreakpointTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerina.devtools.debug;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.ballerinalang.debugger.test.utils.BallerinaTestDebugPoint;
+import org.ballerinalang.debugger.test.utils.DebugTestRunner;
+import org.ballerinalang.debugger.test.utils.DebugUtils;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.eclipse.lsp4j.debug.StoppedEventArguments;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static org.ballerinalang.debugger.test.utils.DebugTestRunner.DebugResumeKind;
+
+/**
+ * Test class for ballerina breakpoints related test scenarios.
+ */
+public class ModuleBreakpointTest extends BaseTestCase {
```

Comment:
Is there any specific reason for moving the `ModuleBreakpointTest` into the distribution?

Reply:
This `ModuleBreakpointTest` is to ensure that debug engage is working fine with standard libraries.

Reply:
Ah I see.. +1